docs/becoming-a-maintainer/translations/es/libro-de-visitas-de-mantenedores.md

```diff
@@ -0,0 +1,13 @@
+# Libro de visitas de  mantenedores
+
+Felicidades por terminar el curso de Conviértete en Mantenedor con OpenSauced. Bienvenido a nuestro libro de visitas de mantenedores, donde puedes compartir tu primer proyecto con nosotros!
+
+## Lista de proyectos
+
+<!-- Use the format below to list your repository -->
```

Comment:
Did you miss this comment to convert? or is this intentional and you are not translating comments? :)

Reply:
Hello Abi, that is correct, the comments should not be translated.